setup.py

```diff
@@ -145,6 +145,9 @@ def initialize_options(self):
 install_requires = []
 extra_kw = {}
 
+compression_source_files = []
```

Comment:
Since these variables are only used in the 'elif', could you move their initialization into that block?